src/main/resources/templates/pps/home.html

```diff
@@ -14,105 +14,97 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 class="govuk-heading-xl" id="main-heading">
-                    Pay a penalty for filing your company accounts late
+                    Pay a penalty to Companies House
                 </h1>
-                <p class="govuk-body-l">Use this service to pay
-                    a company's late filing penalty.</p>
-
+                <p class="govuk-body-l">Use this service to make an online payment by credit or debit card if you've
+                    received a penalty notice from Companies House.</p>
                 <p class="govuk-body">You'll need:</p>
                 <ul class="govuk-list govuk-list--bullet" id="list-requirements">
-                    <li>the company number</li>
-                    <li>the reference number on the penalty notice</li>
-                    <li>to sign in to or create a Companies House account, which is different from a WebFiling account</li>
+                    <li>to sign in to or create a Companies House account (this is different to a WebFiling account)
+                    </li>
                     <li>a credit or debit card</li>
+                    <li>the information on your penalty notice (that we sent you in the post)</li>
                 </ul>
 
+                <div class="govuk-!-margin-top-6">
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text">I've lost my penalty notice
+                            </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <p class="govuk-body"> Email <a href="mailto:enquiries@companieshouse.gov.uk"
+                                                            class="govuk-link">enquiries@companieshouse.gov.uk</a> to
+                                let us know if you have lost your penalty notice. Use 'Lost penalty notice' as the email
+                                subject.
+                            </p>
+                            <p></p>Tell us as much information as you can about the penalty so we can pass your message
+                            to the right team.
+                        </div>
+                    </details>
+                </div>
+
+                <p class="govuk-body">You must pay the full amount.</p>
+                <p class="govuk-body">It takes around 5 minutes to pay.</p>
+
+                <a href="what-does-the-penalty-reference-start-with" role="button" draggable="false"
+                   class="govuk-button govuk-button--start" data-module="govuk-button" data-event-id="pps - Start now"
```

Comment:
On the data-event-id, this stuff will have an overhaul as part of the Matomo configuration so we can look at this further at that point, will need to plumb in this sort of id tracking for more than just a single button too.